src/docker-in-docker/install.sh

```diff
@@ -350,6 +402,7 @@ if [ "${INSTALL_DOCKER_BUILDX}" = "true" ]; then
     chown -R "${USERNAME}:docker" "${docker_home}"
     chmod -R g+r+w "${docker_home}"
     find "${docker_home}" -type d -print0 | xargs -n 1 -0 chmod g+s
+    set +x # for getting detailed logs in case of a failure - flag to mark end of code
```

Comment:
Remove? (as we are removing other block of code)

Reply:
removed 👍